src/alchemlyb/tests/parsing/test_amber.py

```diff
@@ -54,24 +107,6 @@ def test_u_nk_improper(improper_filename,
     """Test the u_nk has the correct form when extracted from files"""
     try:
         u_nk = extract_u_nk(improper_filename, T=300)
-
         assert u_nk.index.names == names
-
-    except:
+    except Exception:
```

Comment:
Can we catch a specific exception here, instead of catch all?

Reply:
I added ```Exception```, but before it was just ```except```, I tried to catch a more specific exception, but I finished in a loop-hole of ```raise```/```except``` that lead to issue #227 .

Reply:
For future work, it might be good to not shuffle a test if it is not necessary as it could make diff hard to read. I will be wondering why you need to remove this test and why you need to add a test. If you want to shuffle the order, it might be good to do it in a separate PR then I will know that there is no code change and it is just shuffling the order.